app/controllers/theblog/application_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Theblog
   class ApplicationController < ActionController::Base
+    include Theblog::ApplicationHelper
```

Comment:
mb it's better to define some methods inside the application_controller with helper_method?